sbirez/templates/sbirez/cost_volume.html

```diff
@@ -37,7 +38,7 @@
             <dt class="topic-number">Topic number</dt>
             <dd class="topic-number">{{ prop.topic.topic_number }}</dd>
             <dt class="total">Total dollar amount for this proposal</dt>
-            <dd class="total money">{{ data.cv_totals.estimated_total_cost|default_if_none:"" }}</dd>
+            <dd class="total money">${{ data.cv_totals.estimated_total_cost| default_if_none:"" | money}}</dd>
```

Comment:
I was thinking there's probably a way to work a dollar sign into the stylesheet when the class is "money".  But we can do this for now and clean up the template later if we find one.